### PR TITLE
Add video example

### PR DESCRIPTION
**Pull request by mfsilva22**
Added a question to exampleCourse to illustrate:

- how to add embedded videos
- how to use `pl-multiple-choice` for a survey question, where there is no correct answer

The question was added to the demo assessment. 

image

### CODE REVIEW

File: exampleCourse/questions/demo/insertEmbeddedVideo/question.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
+  <p>This is an awesome video with an introduction to PrairieLearn.
+     Learn about how to create randomized assessments, visualize your course statistics and more!</p>
```
Comment (nicknytko) [inline exampleCourse/questions/demo/insertEmbeddedVideo/question.html:3]:
```suggestion
     Learn about how to create randomized assessments, visualize your course statistics, and more!</p>
```

I like using serial comma :slightly_smiling_face:

File: exampleCourse/questions/demo/insertEmbeddedVideo/question.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<p>
+  <iframe id="kmsembed-1_e1gprkci" width="640" height="394" src="https://mediaspace.illinois.edu/embed/secure/iframe/entryId/1_e1gprkci/uiConfId/26883701" class="kmsembed" allowfullscreen webkitallowfullscreen mozAllowFullScreen allow="autoplay *; fullscreen *; encrypted-media *" sandbox="allow-forms allow-same-origin allow-scripts allow-top-navigation allow-pointer-lock allow-popups allow-modals allow-orientation-lock allow-popups-to-escape-sandbox allow-presentation allow-top-navigation-by-user-activation" frameborder="0" title="Kaltura Player"></iframe> </p>
+
+</pl-question-panel>
+
+<pl-question-panel><hr></hr></pl-question-panel>
+
+<pl-question-panel>
```
Comment (nicknytko) [inline exampleCourse/questions/demo/insertEmbeddedVideo/question.html:11]:
```suggestion
```

**Comment by nicknytko** [comment]
Is this embedded from mediaspace?  Can we put in a small text blurb saying that you can insert videos by copying the "embed" code on a mediaspace video?

File: exampleCourse/questions/demo/insertEmbeddedVideo/question.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+          <p>
+             <iframe id="kmsembed-1_e1gprkci" width="640" height="394" src="https://mediaspace.illinois.edu/embed/secure/iframe/entryId/1_e1gprkci/uiConfId/26883701" class="kmsembed" allowfullscreen webkitallowfullscreen mozAllowFullScreen allow="autoplay *; fullscreen *; encrypted-media *" sandbox="allow-forms allow-same-origin allow-scripts allow-top-navigation allow-pointer-lock allow-popups allow-modals allow-orientation-lock allow-popups-to-escape-sandbox allow-presentation allow-top-navigation-by-user-activation" frameborder="0" title="Kaltura Player"></iframe>
+          </p>
+      </div>
+  </div>
+
+  <div class="card my-2">
+      <div class="card-header">
+          This example demostrates how you can use <code>pl-multiple-choice</code> for survey type of questions, where
```
Comment (nicknytko) [inline exampleCourse/questions/demo/insertEmbeddedVideo/question.html:20]:
```suggestion
          This example demonstrates how you can use <code>pl-multiple-choice</code> for survey types of questions, where
```

File: exampleCourse/questions/demo/insertEmbeddedVideo/question.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          there is no correct answer.
+          You can use similar approach for <code>pl-checkbox</code>.
```
Comment (nicknytko) [inline exampleCourse/questions/demo/insertEmbeddedVideo/question.html:22]:
```suggestion
          You can use a similar approach for <code>pl-checkbox</code>.
```

File: exampleCourse/questions/demo/insertEmbeddedVideo/question.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<pl-question-panel>
+
+  <div class="card my-2">
+      <div class="card-header">
+          An example to show how you can embed a video in a question. For example, if your video is on MediaSpace, you can go to <code>Share</code>, then
+          <code>Embed</code> and copy the text from there.
+      </div>
+      <div class="card-body">
+          <p> This is an awesome video with an introduction to PrairieLearn.
+               Learn about how to create randomized assessments, visualize your course statistics and more!</p>
```
Comment (nicknytko) [inline exampleCourse/questions/demo/insertEmbeddedVideo/question.html:10]:
```suggestion
               Learn about how to create randomized assessments, visualize your course statistics, and more!</p>
```

**Comment by mfsilva22** [comment]
> hmm, it looks like those old commits came back when you pushed again. i can undo them but we should figure out how they're sneaking their way in 😮

I'm glad I checked with you before! I knew there was something weird going on :-)

**Comment by nicknytko** [comment]
I've merged this back with master so we should be okay now.  Just left some minor comments on the wording.

**Pull request merged**